package.json

```diff
@@ -5,30 +5,21 @@
     "license": "CC-BY-3.0",
     "repository": "https://github.com/fluid-project/infusion-docs",
     "scripts": {
-        "docpad": "node node_modules/docpad/bin/docpad run --env static --out out",
-        "deploy": "node node_modules/docpad/bin/docpad deploy-ghpages --env static",
-        "pretest": "node node_modules/docpad/bin/docpad generate --env static --out out",
+        "11ty": "mkdir -p out && eleventy --serve",
+        "build": "mkdir -p out && eleventy",
```

Comment:
`mkdir` does not perform consistently across operating systems (i.e. Windows).

Use npm package `rimraf` instead to create the directory if needed. See this as an example: https://github.com/fluid-project/floeproject.org/blob/53159c4e8db1e5705e3d0e54569e353be0f2da0b/package.json#L8

Also, it would be good to have the npm scripts be consistent with other IDRC web projects (i.e. `start` `build` and `clean` scripts). For example:
* [IDRC website package.json](https://github.com/inclusive-design/idrc/blob/dev/package.json)
* [Floe Project Website package.json](https://github.com/fluid-project/floeproject.org/blob/master/package.json)

Reply:
I have now used `mkdirp` to do the work of `mkdir -p`.

I have now renamed the scripts as per the two references given by you. `clean` uses the `rimraf` package, whereas `start`, `serve` and `build` don't need it since `mkdirp` overwrites if the directory is already present.